apps/web-giddh/src/app/shared/header/header.component.scss

```diff
@@ -598,11 +604,11 @@ ul.sub-list {
     max-height: 290px;
 }
 
-.upgradePlan.list-inline > li {
+.upgrade-plan.list-inline > li {
     padding-left: 0 !important;
 }
 
-.nav-left-bar .company-name .cText span {
+.nav-left-bar .company-name .c-text span {
```

Comment:
if we update the naming convention of class so we need to update class as a generic name c-text is not a generic name, update class name

Reply:
generic class name will be covered in code designing